config/kubernetes/production/deployment.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         - name: webapp
           image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/correspondence/track-a-query-ecr:latest
           ports:
-          - containerPort: 3000
+            - containerPort: 3000
```

Comment:
Just checking, was this additional whitespace intended?

Reply:
It was, just to make the yaml consistent between envs.